src/commands/deposit/index/processor.ts

```diff
@@ -93,16 +93,21 @@ export async function deposit(
     thumbnail: `attachment://${qrFileName}`,
     description: `Below is the deposit address linked to your Discord account. Please copy your deposit address and paste it into your third-party wallet or exchange.\n\n*Please send only **${symbol}** to this address.*\n\n${getEmoji(
       "CLOCK"
-    )} Your deposit address is **only valid for 3 hours**.\n\n**${symbol} Deposit Address**\n\`\`\`${
-      data.contract.address
-    }\`\`\``,
+    )} Your deposit address is **only valid for 3 hours**.\n\n**${getEmoji(
+      "ANIMATED_POINTING_DOWN",
+      true
+    )} ${symbol} Deposit Address (${data.contract.chain.name})**${getEmoji(
```

Comment:
chain should be option since our data is not 100% correct
data.contract.chain?.name